lib/content_encoding.c

```diff
@@ -851,6 +851,13 @@ static const struct Curl_cwtype * const encodings[] = {
   NULL
 };
 
+/* supported content decoders only for transfer encodings */
+static const struct Curl_cwtype * const transfer_decoders[] = {
```

Comment:
Wouldn't it be easier to keep a single table and add bit flags in the `Curl_cwtype` structure to specify the allowed phase(s) and if it may be skipped or not? Just a suggestion.

Reply:
I prefer to keep them separate and void testing a flag while iterating. Matter of taste, probably.